lib/cartopy/tests/crs/helpers.py

```diff
@@ -11,4 +11,4 @@
 def check_proj_params(name, crs, other_args):
     expected = other_args | {f'proj={name}', 'no_defs'}
     proj_params = set(crs.proj4_init.lstrip('+').split(' +'))
-    assert expected == proj_params
+    assert expected == proj_params, expected.symmetric_difference(proj_params)
```

Comment:
I think this might be less informative because it would just print the string of the values that were different but you wouldn't know which one was in the left vs right?

Reply:
Sorry, I forgot to remove this...